packages/core/src/js/feedback/FeedbackFormScreen.tsx

```diff
@@ -0,0 +1,102 @@
+import { captureFeedback } from '@sentry/core';
+import type { SendFeedbackParams } from '@sentry/types';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import { Alert, Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import defaultStyles from './FeedbackFormScreen.styles';
+import type { FeedbackFormScreenProps, FeedbackFormScreenState } from './FeedbackFormScreen.types';
+
+/**
+ * Implements a feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
+ */
+export class FeedbackFormScreen extends React.Component<FeedbackFormScreenProps, FeedbackFormScreenState> {
+  public constructor(props: FeedbackFormScreenProps) {
+    super(props);
+    this.state = {
+      name: '',
+      email: '',
+      description: '',
+    };
+  }
+
+  public handleFeedbackSubmit: () => void = () => {
+    const { name, email, description } = this.state;
+    const { closeScreen, text } = this.props;
+
+    const trimmedName = name?.trim();
+    const trimmedEmail = email?.trim();
+    const trimmedDescription = description?.trim();
+
+    if (!trimmedName || !trimmedEmail || !trimmedDescription) {
+      const errorMessage = text?.formError || 'Please fill out all required fields.';
```

Comment:
Let's extract all the constant text to constants.

Reply:
Good idea 👍 Updated with 793475679677fde090d48686116c7346dde51663